test/Microsoft.AspNetCore.Blazor.Build.Test/TestFiles/DesignTimeCodeGenerationTest/ChildComponent_WithChildContent/TestComponent.ir.txt

```diff
@@ -0,0 +1,36 @@
+Document - 
```

Comment:
This format ends up being the most useful for understanding bugs in Razor. This is the main reason why I wanted to switch to this style. It ends up being faster and easier to just add a new test and run it than to debug.

Reply:
Sounds good!